hardwareModules/RoFICoM/software/control_board/src/bsp.cpp

```diff
@@ -0,0 +1,124 @@
+#include "bsp.hpp"
+
+#include "configuration.hpp"
+#include "motor.hpp"
+
+#include <optional>
+
+#include <stm32g0xx_hal.h>
+#include <stm32g0xx_ll_rcc.h>
+#include <stm32g0xx_ll_system.h>
+#include <stm32g0xx_ll_cortex.h>
+#include <stm32g0xx_ll_utils.h>
+#include <stm32g0xx_ll_i2c.h>
+
+namespace {
+    // TODO ? <stm32cxx/drivers/clock.hpp>
```

Comment:
Co má tento komentář říci? Že chceme do STM32CXX implementovat nastavení hodin? Tomu bych se asi vyhl, protože nastavení hodin je hodně procesor-specifická věc a nějaká abstrakce se buduje složitě.